storybook/custom-controls/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './currency';
+
+export const INTERACTION_TIMEOUT = 3500;
```

Comment:
I think you may have forgotten to change this to something lower here ;)

Reply:
Thanks for catching that! 🤦‍♀️